bundles/org.openhab.binding.vesync/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -242,14 +263,14 @@
 	</channel-type>
 
 	<channel-type id="deviceAFConfigAutoPrefRoomSizeType">
-		<item-type>Number:Dimensionless</item-type>
+		<item-type unitHint="one">Number:Dimensionless</item-type>
 		<label>Config: Room size</label>
 		<description>Room size (foot sq) for efficient auto mode</description>
 		<state readOnly="true" pattern="%.0f sq ft"/>
```

Comment:
```suggestion
		<state readOnly="true" pattern="%.0f %unit%"/>
```

Reply:
This need to be adapted anyway. As this should also be a quantitytype with area dimension

Reply:
Sure I can update - this one. It would potentially break anyone with this brought in (although I doubt many people use it). If is done and is a breaking change it's tempting to do the other clean-up's to Number from Number:Dimesnionless at the same time. As if there's one you may as well do the lot in a single pass I guess?

Reply:
Updated just this one - for some reason I thought it was under the humidifier as well but it isn't as its only relevant to the air filters auto mode.